### DISCUSSION TITLE
How can i get subscription paymentState?

### DISCUSSION THREAD

**Discussion by SlavaSt295** [imdhemy/laravel-in-app-purchases#56] [Q&A] [answered]
Hello

Is it posible to get paymentState property from subscription response? 

![image](https://user-images.githubusercontent.com/38761838/115629450-0a6fd100-a30b-11eb-9120-724539a41480.png)

Thanks!

**Comment by imdhemy** [comment, marked as answer]
@SlavaSt295,

Just make sure you are using `imdhemy/google-play-billing` of version `0.6.3` or later. 

Run 
```
composer show imdhemy/google-play-billing
```
And check the `versions` value.

Otherwise, just run 

```
composer update
```

Then you can access the `getPaymentState()` method.

**Reply by SlavaSt295** [reply]
You are the best, thanks!
